config/application.yml

```diff
@@ -0,0 +1,2 @@
+gmail_username: 'hypertube.thewildcards@gmail.com'
```

Comment:
Ok, so I should just delete this file?

And I'm not entirely sure what **username and password** I'm supposed to **re-add**? 🤔 

<img width="623" alt="screen shot 2017-04-28 at 9 58 08 am" src="https://cloud.githubusercontent.com/assets/18417244/25519784/82b7bd40-2bf9-11e7-867c-6006384f3e3c.png">